.github/ISSUE_TEMPLATE/begin-the-hunt.yml

```diff
@@ -11,17 +11,16 @@ body:
     validations:
       required: true
 
-  - type: textarea
+  - type: checkboxes
```

Comment:
This way the layout cannot be messed with accidentally when submitting issue. The parsing of the body relies on the layout to be as is!